sites/all/modules/custom/all_search_page_configuration/all_search_page_configuration.pages_default.inc

```diff
@@ -444,10 +445,10 @@ function all_search_page_configuration_default_page_manager_pages() {
   $pane->access = array();
   $pane->configuration = array(
     'admin_title' => 'Results section anchor links',
-    'title' => '',
-    'body' => '<div class="results-section"><label class="show-result-section-menu" for="show-result-section-menu">Skip to Section</label><input id="show-result-section-menu" type="checkbox" /><ul id="result-section-menu" class="result-section-menu"><li><a href="#catalog_block-catalog_blacklight_results">Catalog</a></li><li><a href="#summon_block-summon_search_results">Articles+</a></li><li><a href="#databases_keyword_search-panel_pane_2">Databases</a></li><li><a href="#catalog_block-catalog_pulfa_results">Library Archives</a></li><li><a href="#catalog_block-catalog_pudl_results">Digital PUL</a></li><li><a href="#catalog_block-catalog_arts_results">Art Museum Collections</a></li><li><a href="#catalog_block-catalog_pulmap_results">Maps and Geographic Data</a></li><li><a href="#general_site_keyword_search-panel_pane_1">Library Website</a></li><li><a href="#general_site_user_search-panel_pane_1">Library Staff</a></li><li><a href="#libguides_block-libguides_search_results">Library Guides</a></li><li><a href="#catalog_block-catalog_faq_results">Library FAQs</a></li></ul></div>',
+    'body' => '<div class="results-section"><label class="show-result-section-menu" for="show-result-section-menu">Skip to Section</label><input id="show-result-section-menu" type="checkbox" /><ul id="result-section-menu" class="result-section-menu"><li><a href="#catalog_block-catalog_blacklight_results">Catalog</a></li><li><a href="#summon_block-summon_search_results">Articles+</a></li><li><a href="#databases_keyword_search-panel_pane_2">Databases</a></li><li><a href="#catalog_block-catalog_pulfa_results">Library Archives</a></li><li><a href="#catalog_block-catalog_pulfalight_results">Library Archives</a></li><li><a href="#catalog_block-catalog_pudl_results">Digital PUL</a></li><li><a href="#catalog_block-catalog_arts_results">Art Museum Collections</a></li><li><a href="#catalog_block-catalog_pulmap_results">Maps and Geographic Data</a></li><li><a href="#general_site_keyword_search-panel_pane_1">Library Website</a></li><li><a href="#general_site_user_search-panel_pane_1">Library Staff</a></li><li><a href="#libguides_block-libguides_search_results">Library Guides</a></li><li><a href="#catalog_block-catalog_faq_results">Library FAQs</a></li></ul></div>',
```

Comment:
You will want to remove `<li><a href="#catalog_block-catalog_pulfa_results">Library Archives</a></li>` in favor of the new pulfalight results option.